src/libs/reportUtils.js

```diff
@@ -142,12 +142,13 @@ function findLastAccessedReport(reports, ignoreDefaultRooms) {
 /**
  * Whether the provided report is an archived room
  * @param {Object} report
+ * @param {String} report.chatType
  * @param {Number} report.stateNum
  * @param {Number} report.statusNum
  * @returns {Boolean}
  */
 function isArchivedRoom(report) {
-    if (!isDefaultRoom(report)) {
+    if (!isChatRoom(report) && !isPolicyExpenseChat(report)) {
```

Comment:
Ah I see why you asked about the policyRooms being archivable.  I think this is fine. Hypothetically any "chatRoom" should be archivable. If you wanted to be really strict you could keep it just defaultRooms for now but this is fine.